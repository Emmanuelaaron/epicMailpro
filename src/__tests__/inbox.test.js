import React from 'react';
import { shallow } from 'enzyme';
import InboxComponent from '../components/inbox/index.js';
describe('<InboxComponent />', () => {
  const props = {
    record: {
      body: 'sleek',
    },
  };
  it('it should render component without fail', () => {
    const wrapper = shallow(<InboxComponent {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});