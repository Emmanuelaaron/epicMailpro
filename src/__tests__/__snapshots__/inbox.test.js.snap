// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InboxComponent /> it should render component without fail 1`] = `
<div>
  <div
    className="header"
  >
    <a
      className="logo"
      href="#"
    >
      <span
        id="whitish"
      >
        Epic
      </span>
      <span
        id="blueish"
      >
        Mail
      </span>
    </a>
    <div
      className="header-right"
    >
      <a
        href=""
      >
        Logout
      </a>
    </div>
  </div>
  <div
    className="dashboard-content"
  >
    <div
      className="side-bar"
    >
      <br />
      <a>
        Compose
      </a>
      <br />
      <br />
      <a>
        Inbox
      </a>
      <br />
      <br />
      <a>
        Sent
      </a>
      <br />
      <br />
      <a>
        Groups
      </a>
      <br />
      <br />
      <a>
        Create Group
      </a>
    </div>
    <div
      className="messages"
    >
      <h1>
        My messages
      </h1>
      <hr />
      <p
        id="no-messages"
      />
    </div>
  </div>
</div>
`;
